src/shared/src/backend-supports/projects-get-all.ts

```diff
@@ -12,7 +12,12 @@ const calculateEndDate = (start: Date, weeks: number) => {
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+interface WP {
```

Comment:
Also, can we locate this interface before the calculateEndDate function? For our file formatting, we wanna place the interface in the beginning after the imports.